blog/2020-07-31-temporal-v0.28.0-changelog.md

```diff
@@ -0,0 +1,111 @@
+---
+tags:
+- changelog
+- temporal
+- v1
+posted_on_: 2020-07-31T00:00:00
+id: temporal-v0.28.0-changelog
+title: Temporal v0.28.0 changelog
+author: Cully Wakelin
+author_title: SDE Technical Writer
+author_image_url: https://avatars2.githubusercontent.com/u/34380806?s=400&u=5cd38b5e4416a5d10cdf9ebd386eec1d02f0b067&v=4
+release_version: v0.28.0
+---
+
+<!--truncate-->
+
+Today is an exciting day for us, as Temporal is releasing its newest version, v0.28.0, which brings us closer to a stable production ready release. We thank all of the users who have helped us iterate and move the product forward.
+
+This changelog includes all of the changes since Temporal forked from Cadence, and should help users who are interested in upgrading.
+
+## What has changed?
+
+- **Cadence Oct 2019** → **Temporal v0.28.0 Alpha**: The software version has bumped.
+- **TChannel/Thrift** → **gRPC/Protobuf**:
+    - Server Thrift API → Protobuf based Go API (go.temporal.io/api).
+    - All communication over the wire can be encrypted via [TLS](https://docs.temporal.io/docs/configuring-temporal-server#tls). Note: Support for TLS in tctl and temporal-web is still in-progress.
+    - Thrift errors have been converted to custom service errors.
+    - Standard gRPC health service is used for health checks.
+    - Removed Uber YARPC library.
```

Comment:
I think it worth to add between 28-29:
 - Standart protobuf types are used for times and durations.